sdk/src/contexts/abstracts/tokenBridge.ts

```diff
@@ -38,6 +38,9 @@ export abstract class TokenBridgeAbstract {
   protected abstract formatAddress(address: string): any;
   protected abstract parseAddress(address: any): string;
 
+  protected abstract formatAssetAddress(address: string): Promise<any>;
+  protected abstract parseAssetAddress(address: any): Promise<string>;
```

Comment:
Is it different for account addresses?

Reply:
Yes, Sui account addresses are 32 byte hex strings so we don't even need to parse/format them since they're already "formatted". But assets/tokens are different - e.g. the gas token on Sui is `0x2::sui::SUI` and to convert it to a wormhole address (and vice versa) we have to look it up from the Sui token bridge contract unfortunately (that's why these methods are `async`)